### ISSUE TITLE
Genshin Impact: graphical bug and driver crashes

### ISSUE THREAD

**Issue by BiZkViT96** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#449]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Genshin Impact 4.0

### Game Platform [Required]

- [ ] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [X] Other (please specify)

### Other game platform

Dedicated installer

### Processor / Processor Number [Required]

i5-12500

### Graphic Card [Required]

UHD 770

### GPU Driver Version [Required]

- [X] 31.0.101.4644
- [ ] 31.0.101.4578
- [ ] 31.0.101.4577
- [ ] 31.0.101.4576
- [ ] 31.0.101.4575
- [ ] 31.0.101.4514
- [ ] 31.0.101.4502
- [ ] 31.0.101.4499
- [ ] 31.0.101.4382
- [ ] 31.0.101.4369
- [ ] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311
- [ ] 31.0.101.4257
- [ ] 31.0.101.4255
- [ ] 31.0.101.4148
- [ ] 31.0.101.4146
- [ ] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12409791/igcit_ssu.txt)

### Description and steps to reproduce [Required]

The problem with crashing the GPU driver for a long time looked completely random to me. Driver could crash after 10 minutes of the game or after 10 hours. But finally I came across a place where this happens 90% of the time. And the driver not only crashes, but also displays a graphical bug. Is this related to all previous crashes I don't know, but this is at least looks like can be debugged.

 I'm not sure, but it looks like the bug occurs only during the day so you have to change time in the menu. Teleport to this statue
![GenshinImpact_2023_08_22_21_06_41_459](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/88586431/8e47bf97-70c8-41c4-aa49-71754e7411dc)

Go to the right side of the map (blue marker with two exclamation marks). You will see sand waterfalls. Specifically in this place they are displayed with a bug. Also GPU driver will crash shortly after you see them.
![GenshinImpact_2023_08_22_20_56_04_072](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/88586431/65ddf746-d5ef-46f0-9db7-14fd17d740ca)

Sometimes everything is displayed as it should
![GenshinImpact_2023_08_22_21_14_35_611](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/88586431/d18296e5-c0c4-48b0-8cc8-5086cb258a96)

![GenshinImpact_2023_08_22_21_23_01_143](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/88586431/36295bf5-2ae5-4e31-9939-1e1be01ab4cc)

Teleport back to the statue and restart the game (exit to the launcher) if the bug/driver crash doesn't happen.

I'm not sure if this map area is changing with the quest but if you don't see this hole then you should check sand waterfalls on opposite side of the canyon they will have the same bug. You can see one of them on top in the last screenshot its little bit to the right from the center.

### Game graphic quality [Required]

- [X] Low
- [ ] Medium
- [ ] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

Off

### Game display mode [Required]

- [X] Fullscreen
- [ ] Windowed
- [ ] Borderless Windowed

### Detailed game settings [Required]

![GenshinImpact_2023_08_22_21_05_53_685](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/88586431/ee2c0808-beac-4370-b620-6776378ec28a)

### Crash dumps [Required, if applicable]

[compressed_dumps.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12409865/compressed_dumps.zip)

**Comment by BiZkViT96** [comment]
I contacted HoYoverse about this issue and they asked to record a video. So why not publish it here as well, here it is 

https://youtu.be/If3RwnIZIm0

**Comment by BiZkViT96** [comment]
Requested by HoYoverse.

[output_log.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12437683/output_log.txt)
[Temp.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12437686/Temp.zip)

**Comment by BiZkViT96** [comment]
If someone from Intel tech support is interested, I've been told that all information has been forwarded to the developers for further investigation. 

So if they able to reproduce the bug they will either fix it on the game side or contact the Intel GPU department. I guess.

**Comment by BiZkViT96** [comment]
Additional crash logs for HoYoverse.

[output_log.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12445427/output_log.txt)
[mihoyocrash_0f4a3736a9e96a4389c96a55514a6e63.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12445428/mihoyocrash_0f4a3736a9e96a4389c96a55514a6e63.zip)

**Comment by Arturo-Intel** [comment]
@BiZkViT96 Thanks for reporting this to the developer too.
We are working on our side to root cause this issue. 

--r2

**Comment by BiZkViT96** [comment]
@Arturo-Intel Can you confirm that HoYoverse have contacted Intel engineers? Because they told me "our colleagues are working on fixing the thing". So I believe they have reproduced the bug.

**Comment by BiZkViT96** [comment]
@Arturo-Intel  As I understand it, you have no updates. I'm still experiencing these random driver crashes.

Here is fresh crash on 31.0.101.4885 driver.
[output_log.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12893844/output_log.txt)
[mihoyocrash_517cd66cdd7819408735d428f0491252.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12893843/mihoyocrash_517cd66cdd7819408735d428f0491252.zip)

**Comment by Arturo-Intel** [comment]
> Can you confirm that HoYoverse have contacted Intel engineers?

No, I can't, I don't have that access level :p, sorry

> So I believe they have reproduced the bug.

I was able to replicate in our labs.

The only thing I can confirm you is our engineers are working on this.
I will share any news here.

Thank you for your patience,
--r2

**Comment by BiZkViT96** [comment]
@IGCIT Replicated.

**Comment by Arturo-Intel** [comment]
@BiZkViT96 are you were able to reproduce this behavior using .5081?

**Comment by BiZkViT96** [comment]
@Arturo-Intel Hi! Thanks for not abandoning this issue. Did you got information that fix is supposed to be in 5081 driver? I have tried several times and I didn't encounter graphical artifacts or driver crashes. Because random nature of this issue I don't know if its has been fixed or I was just lucky. In couple days there is going to be 4.3 update. Let me test on that version to be sure.

**Comment by Arturo-Intel** [comment]
I asked because I was unable of reproduce the issue. Lets wait for 4.3 update to see if this behavior is happening.
-- r2

**Comment by BiZkViT96** [comment]
@Arturo-Intel I am happy to report that after excessive (i think) testing, I cannot reproduce the problem in version 4.3. I'm not sure which side is responsible for fixing the bug, but stull, thank you for your time and help.

**Comment by BiZkViT96** [comment]
@Arturo-Intel  Nope, random crashes is not fixed. Game just crashed driver on Ruin Serpent boss. It looks like there is two separate issues, one with sand (fixed by HoYoverse i think) and other that caused by SFX from characters abilities or from that  boss.

At the moment when the driver crashed, these character abilities were active: Zhongli, Yae Miko, Tighnari. But i think crash was caused by black goo that boss is produced.

[output_log.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13754400/output_log.txt)
[mihoyocrash_6c957affbce7cb4390f3b7b9474155f2.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/13754399/mihoyocrash_6c957affbce7cb4390f3b7b9474155f2.zip)

**Comment by BiZkViT96** [comment]
@Arturo-Intel After full PC reboot and with no MSI Afterburner running in background. I managed to reproduce the bug with only  Barbara in the team. You can get her for free. So this is not a specific character bug. I'm pretty sure the crash is caused by the texture of dust and sand that the game uses when Ruin Serpent hides underground and emerges again. You can see the texture I'm talking about in the screenshot in the post above.

If you are able to reproduce the driver crash, please let me know.

**Comment by BiZkViT96** [comment]
Today, the driver (31.0.101.5186) crashed again, right in the middle of a dialogue with this NPCs.
[output_log.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14047834/output_log.txt)
[mihoyocrash_7c974af1c8c4fb41b67dcbabe9cd6239.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/14047848/mihoyocrash_7c974af1c8c4fb41b67dcbabe9cd6239.zip)

![GenshinImpact_2024_01_25_10_28_44_391](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/88586431/7bbb79fd-8ea7-4ea6-a82d-528cd89f8fb1)

Unfortunately, after I spent a lot of time trying to reproduce the driver crash on boss that I mentioned in the post above, I was unable to do so.

There is 64 million active players, and Im pretty sure that there is at least 50 thousand people who experiences Intel driver crashes in this game on almost daily basis.

**But**

Due to the random nature of this error and lag in receiving any response from Intel support team, I am abandoning this issue.

There is literally nothing I can bring to the table to fix this issue.

@Arturo-Intel you have a two ways here: contact the devs with all information that I provided, so you can dig this further, or just close the issue.

**Comment by Arturo-Intel** [comment]
@BiZkViT96 Due the random nature of this issue (as you mention above), this issue has been hard for me to push it through the process, but it is in the queue, which is good. 

>There is 64 million active players, and Im pretty sure that there is at least 50 thousand people who experiences Intel driver crashes in this game on almost daily basis.

For the above reason, I will keep this issue open, a lot of users are experimenting these random crashes (including myself), and it's not fun at all.

Thank you for all the info you provide, If I have any news, I will let you know through this channel.

-- r2

**Comment by BiZkViT96** [comment]
All logs and data are up-to-date, UHD 770, driver 32.0.101.5768.

Settings

![GenshinImpact_2024_08_02_18_13_20_012](https://github.com/user-attachments/assets/c5c9ee71-0946-4b6d-9685-861b9e45da4f)
![GenshinImpact_2024_08_02_18_13_26_127](https://github.com/user-attachments/assets/4a4cccec-b428-42f4-b3fa-fa0e0218fa78)


Сrash log for HoYoverse.

[output_log.txt](https://github.com/user-attachments/files/16470591/output_log.txt)
[mihoyocrash_565e785213e2e44e856b885b58375123.zip](https://github.com/user-attachments/files/16470607/mihoyocrash_565e785213e2e44e856b885b58375123.zip)
[SSU.txt](https://github.com/user-attachments/files/16470609/SSU.txt)

Graphical bug with sand waterfall.

![GenshinImpact_2024_08_02_18_56_58_123](https://github.com/user-attachments/assets/e0f5fff4-0c01-4d25-a8e8-684d16dc6faa)

https://github.com/user-attachments/assets/40a28e7d-45b9-4a57-a49b-87f940968136

![GenshinImpact_2024_08_02_18_57_24_666](https://github.com/user-attachments/assets/a2f53a0f-67f2-4820-83dd-3617a7dfd658)

https://github.com/user-attachments/assets/e76aed7a-8fec-4518-a24d-b00546fcdfa1

https://github.com/user-attachments/assets/79b24758-24b2-434b-b76f-9569815f897f


@Arturo-Intel Hi! I have reached out to HoYo again. If you still on it, you may want to try replicating the graphical bug mentioned above.

**Comment by BiZkViT96** [comment]
@Arturo-Intel I have some news, it turns out that the graphic artifacts come from RTSS. So forget about them. I removed the mention of them from the message.

**Comment by BiZkViT96** [comment]
@Arturo-Intel For a moment, I thought the artifacts were being caused by RTSS, but they are actually there without it. To see them, you need to press ALT+TAB. The artifacts will be a different color than with RTSS.

**Comment by BiZkViT96** [comment]
New crash log for HoYoverse.
[crashlog.zip](https://github.com/user-attachments/files/16487660/crashlog.zip)


Crash happened on this boss. Screenshot provided for illustrative purposes.

![GenshinImpact_2024_08_04_20_13_07_685](https://github.com/user-attachments/assets/52867da8-80ee-4c2f-a1b7-2e86827f8ec0)

**Comment by rmorera-intel** [comment]
Hello @BiZkViT96 please re-test with new driver [32.0.101.6130](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html). 
By the way, have you received any response from the game developer?

**Comment by BiZkViT96** [comment]
Hi @rmorera-intel ! Thank you for your attention to this issue. Graphical bug after minimizing the game is still present on 101.6130 driver. As for the driver crashes, 101.6130 is too new, but on the previous version 101.6129, I had 4 or 5 crashes last week.

I asked tech support about the status of the issue. Answer I got in summary: Your message has been forwarded to our colleagues from the technical department for the proceedings. Unfortunately, sometimes it takes time to review and solve a problem.

So, basically, there are no changes.

**Comment by BiZkViT96** [comment]
Driver 6297 just crashed... 😒

**Comment by SavageWaffle** [comment]
Issue ongoing fix TBD

**Comment by Felipe-Intel** [comment]
hello @SavageWaffle please use phrases like "Are there updates about this issue?" to avoid confusion in the community. This way the support team can update the ticket accordingly. 
@Arturo-Intel  please check.

**Comment by BiZkViT96** [comment]
Updated to 6632 Beta driver.

Graphical bug after minimizing is still present.

https://github.com/user-attachments/assets/4157fe59-9fb6-4c3b-ac16-a6313fbbec3e

Couple days ago I had a crash on 6559 WHQL driver.

So, yeah. Everything is the same. :disappointed:

**Comment by BiZkViT96** [comment]
@Arturo-Intel I know that driver crashes is very hard to reproduce, but can you atleast confirm graphical bug?  I am able to easily reproduce the issue on version 6723 of the driver.

**Comment by BiZkViT96** [comment]
I have upgraded to B580 dGPU and am no longer able to reproduce graphical bug. So it seems that this issue is exclusive to iGPU. Due to this, I have decided to abandon the issue. Feel free to do whatever you want.

**Comment by rmorera-intel** [comment]
@BiZkViT96 we are going to close this bug now, if there are further reports of this game crashing on iGfx from other users it is best to open a new bug report.
